apps/web/src/pages/integrations/components/ChannelGroup.tsx

```diff
@@ -18,15 +18,51 @@ export function ChannelGroup({
   }
 
   return (
-    <Grid mb={50}>
-      <Grid.Col span={12} data-test-id={`integration-group-${title.toLowerCase()}`}>
+    <Wrapper>
+      <div data-test-id={`integration-group-${title.toLowerCase()}`}>
         <Title size={2}>{title}</Title>
-      </Grid.Col>
-      {providers.map((provider) => (
-        <Grid.Col sm={6} md={3} xs={4} key={provider.providerId}>
-          <ProviderCard provider={provider} onConnectClick={handlerOnConnectClick} />
-        </Grid.Col>
-      ))}
-    </Grid>
+      </div>
+      <ContentWrapper>
+        {providers.map((provider) => (
+          <div key={provider.providerId}>
+            <ProviderCard provider={provider} onConnectClick={handlerOnConnectClick} />
+          </div>
+        ))}
+      </ContentWrapper>
```

Comment:
A small question, I think it would be better to have the native mantine implementation here rather than a custom grid. Any particular reason it was needed, seems you got it to work on your first PR with this approach.

Reply:
Hi.
The first option, using Mantine, works, but it is not smooth enough as the second one. 
So, watch the second and the third video in the issue and respond: which option is the best for you?